nixos/modules/virtualisation/anbox.nix

```diff
@@ -130,7 +135,8 @@ in
             --container-network-gateway=${cfg.ipv4.gateway.address} \
             --container-network-dns-servers=${cfg.ipv4.dns} \
             --use-rootfs-overlay \
-            --privileged
+            --privileged \
+            --daemon
```

Comment:
don't you have to add `Type=fork` if you use daemon here?
You are adding it only to avoid the warning or any other reason?

Reply:
I did add it to avoid the warning but the process doesn't fork anyways, or at least doesn't exit.